posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+This list view allows you to visualize the performance of your DB with several very useful filters. Indeed, you can choose to observe the CPU performance, but also the input/output in terms of data or logs, the duration of the queries and finally the number of execution of the queries, all on time intervals chosen by another filter. Thus, this tool can be very useful when the need to analyze or improve the performance of the DB is felt.
+
+Let's now move to the "Connection Strings" view.
+
+![Figure 12 : View of the Connection Strings page](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_12.png)
+
+This view lists the different connection strings available for C#, C++, Java, PHP, Go and finally the ODBC standard for more openness. Normally, you should see these strings in the boxes but they have been removed for confidentiality reasons. All you have to do is to copy the URL you are interested in into your own DB access configuration and customize your authentication information, the famous user/password!
+
+Now go back to the DB overview page to go to the DB server by clicking on the _srvint_ link.
+
+![Figure 13 : View of a SQL Server](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_13.png)
+
+From this screen, you have the contact of the DB administrator in the "Active Directory Admin" field in case you need support and a quick link back to _dbint_ that redirects you to the DB.
+
+In the left menu there are some useful shortcuts like SQL databases which again allows you to find the link to the DB or _Firewalls and virtual networks_ which allows you to manage the IP addresses allowed to connect to the DB.
+
+![Figure 14 : "SQL databases" view](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_14.png)
+
+This screen lists all the DBs created in the SQL Server you are browsing in order to retrieve the connection strings to the different DBs.
+
+![Figure 15 : "Firewalls and virtual networks" view](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_15.png)
+
+This view allows you to create as many rules as needed to allow single IP addresses or ranges of IP addresses to connect to the DB. Developers can create rules to allow IP address ranges from their workplace to access remote DBs from their own computer.
+
+## Storages
+
+### Overview
+
+![Figure 16 : "Overview of storage accounts" view](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_16.png)
+
+This view is the home page of a storage account. It provides access to the different storage services such as containers, file shares, tables and queues (we will only develop containers and queues). It also provides a quick link to the _Azure Storage_ online documentation.
```

Comment:
This view is the home page of a storage account. It provides access to the different storage services, such as containers, file shares, tables, and queues (we will only develop containers and queues). It also provides a quick link to the _Azure Storage_ online documentation.